instance-scheduler/main.go

```diff
@@ -299,6 +445,9 @@ type InstanceSchedulingResponse struct {
 	ActedUpon             int      `json:"acted_upon"`
 	Skipped               int      `json:"skipped"`
 	SkippedAutoScaled     int      `json:"skipped_auto_scaled"`
+	RDSActedUpon          int      `json:"rds_acted_upon"`
+	RDSSkipped            int      `json:"rds_skipped"`
+	RDSSkippedAutoScaled  int      `json:"rds_skipped_auto_scaled"`
```

Comment:
This is not needed, if not reading autoscaling groups

Reply:
removed autoscaling section